.github/workflows/release-windows.yml

```diff
@@ -0,0 +1,58 @@
+name: Build and Release [windows]
```

Comment:
We're not going to use this for releases, and I'd ask to remove it from the PR please.